python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -3831,6 +3834,50 @@ def callback(self, pre, post, node_map):
         transpose2 = tvm.relay.transpose(transpose1, axes=[0,2,1]) # (n_kv_heads*brcst_val, head_dim, bs*seqlen)
         return transpose2
 
+class RemoveDenseInputSqueeze(DFPatternCallback):
```

Comment:
Can you add brief doc to this class why we're doing this :))

Reply:
Done